### PR TITLE
Rysy

### PR DESCRIPTION
**Pull request by ggruszczynski**
TCLB shall run within singularity container or on bare metal. Your choice ;)
It shall `start` on >1 Node (4>GPU).
See https://github.com/CFD-GO/TCLB_cluster/issues/10

### CODE REVIEW

File: cluster/rysy.bash [hunk 1]
```diff
@@ -27,23 +27,57 @@ fix MODULES_BASE ""
...
+		def CONFOPT "--enable-cpp11 --with-cuda-arch=sm_60"
+		# def RUN_COMMAND "mpirun"
+		def RUN_COMMAND "srun"
 		def MAX_TASKS_PER_NODE 4
 		def MAX_TASKS_PER_NODE_FOR_COMPILATION 24
 		def CORES_PER_TASK_FULL 1
 		def MEMORY_PER_CORE 5
+
 		;;
+	# case ${RUN_GPU}+${RUN_SINGULARITY} in
```
Comment (ggruszczynski) [inline cluster/rysy.bash]: how to do it?

File: config [hunk 2]
```diff
@@ -134,16 +138,25 @@ EOF
 
 if ask_no "Do you want to install needed R packages now?"
 then
-	salloc $(arg -p $DEBUGQ) --ntasks=1 $PP/exec.install skipssl rdep
+	if ARE_U_ON_RYSY
```
Comment (llaniewski) [inline config]: There should not be any cluster specific code outside of `cluster/...` Remember that `arg` function works in that way, that if `$DEBUG_QOS` is empty then `$(arg --qos $DEBUG_QOS)` will be empty (look: https://github.com/CFD-GO/TCLB_cluster/blob/89d1366fb06cf12316845bf79e07c86e00b083ab/lib.bash#L188)

File: checks.bash [hunk 3]
```diff
@@ -34,15 +34,29 @@ function RHOME_CHECK {
...
 function MODULES_CHECK {
-    for i in $@
-    do
-            if [ -z "$(module av $i 2>&1)" ]
-            then
-                    echo there is no module $i
-                    return 1;
-            fi
-    done
+	if ! ARE_U_ON_RYSY
```
Comment (llaniewski) [inline checks.bash]: This should be a general option like "DONT_CHECK_MODULES" or something.

**Review by llaniewski — CHANGES_REQUESTED** [review]

File: config (hunk 2 above)
Comment (ggruszczynski) [reply config]: done

File: checks.bash (hunk 3 above)
Comment (ggruszczynski) [reply checks.bash]: done

**Pull request merged**